package.json

```diff
@@ -14,7 +14,8 @@
     "lint": "eslint '**/*.{js,ts}'",
     "mocha": "nyc mocha --opts mocha.opts",
     "prepare": "npm run clean && npm run build -- --build tsconfig.publish.json",
-    "test": "npm run lint -- --quiet && npm run mocha"
+    "test": "npm run lint -- --quiet && npm run mocha",
+    "document": "rm ./docs/*.md && ts-node scripts/generate-docs.ts"
```

Comment:
Need to change this not to need `ts-node`

Reply:
I don't think you *need* to do that. `ts-node` is a dev dependency and this would only be run with all those dependencies installed. Also, it would allow you to directly import the `src` directory (which I saw you wanted)

Reply:
Also, maybe `build:docs` as the script name?